crates/pixi_default_sys/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+use rattler_conda_types::{Platform, Version};
+
+/// The default GLIBC version to use. This is used when no system requirements are specified.
+pub fn default_glibc_version() -> Version {
+    "2.28".parse().unwrap()
```

Comment:
do we need to make in static?

Reply:
I think the overhead of a OnceLock or some sort is not really worth it :) WDYT @baszalmstra?

Reply:
I guess it would be more correct but I also don't think it matters.